kernel/network/e1000.h

```diff
@@ -176,5 +177,5 @@ extern e1000_device_t* e1000;
 void e1000_init(void);
```

Comment:
Maybe it would be cleaner to have a getter for the e1000 (with a static variable) instead of an exported global variable? (this may also be the case for the arp table)